webapp/src/language/en.json

```diff
@@ -35,7 +35,12 @@
     "missedBlocks": "Missed Blocks",
     "timeFrame": "Time frame",
     "net": "NET",
-    "cpu": "CPU"
+    "cpu": "CPU",
+    "boot": "Boot Nodes",
+    "writer": "Writer Nodes",
+    "observer": "Observer Nodes",
+    "validator": "Validator Nodes",
```

Comment:
`Validator` is only used in LACChain , outside of that the correct term is `Producer`